source/iostreams.tex

```diff
@@ -8805,9 +8806,9 @@
 If the open operation succeeds and
 \tcode{(mode \& ios_base::ate) != 0},
 positions the file to the end
-(as if by calling
-\tcode{fseek(file, 0, SEEK_END)}).\footnote{The macro
-\tcode{SEEK_END}
+(as if by calling \tcode{fseek(file, 0, SEEK_END)}, where
+\tcode{file} is the pointer returned by calling \tcode{fopen}).
```

Comment:
Escape the space at the end of the line; there must be no whitespace before a footnote.